server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/def/Validations.java

```diff
@@ -115,6 +115,7 @@ static String checkDbIdentifier(@Nullable String identifier, String identifierDe
   }
 
   private static void checkDbIdentifierCharacters(String identifier, String identifierDesc) {
+    checkArgument(identifier != null && identifier.length() > 0, "Identifier must not be empty");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2589)